File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        bool ans = false;
        sort(nums.begin(), nums.end());
        for(int i=1; i<nums.size(); i++)
        {
            if(nums[i-1] == nums[i])
            {
                ans = true;
                break;
            }

            else continue;
        }
        return ans;
    }
};

File: Data Structures & Algorithms/k-closest-points-to-origin/submission-0.py
class Solution:
    def kClosest(self, points: List[List[int]], k: int) -> List[List[int]]:
        temp = [[x*x + y*y, x, y] for x, y in points]
        
        temp.sort()
        ans = []

        for i in range(k):
            print(i)
            ans.append([temp[i][1], temp[i][2]])
            
        return ans
